runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TabContentPanel.java

```diff
@@ -78,9 +81,10 @@ protected static String getFormattedEstimate(long remainingSeconds, boolean useR
 			LocalDateTime currentTime = LocalDateTime.now();
 			if (endTime.getDayOfWeek() != currentTime.getDayOfWeek())
 			{
-				sb.append(endTime.getDayOfWeek().getDisplayName(TextStyle.FULL, Locale.getDefault())).append(" ");
+				sb.append(endTime.getDayOfWeek().getDisplayName(TextStyle.FULL, RuneLite.SYSTEM_LOCALE)).append(" ");
```

Comment:
Dont use system locale for day of the week name

Reply:
Replaced with `Locale.ENGLISH`

Reply:
Replaced with `Local.getDefault()` as that was what was there before.